{"name": "test_verify_phone_no[device1-phone_number0]", "status": "passed", "attachments": [{"name": "log", "source": "59e05f82-940f-49a8-9175-77e68a194680-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "appium_driver", "value": "'device1'"}, {"name": "phone_number", "value": "['9021921234']"}], "start": 1638782720096, "stop": 1638782727411, "uuid": "5f983692-560e-4e82-9c6e-adcce7f71c6e", "historyId": "263255fd6f2ba9ad306038ea95fbb885", "testCaseId": "8f179043caacb0abcb24fe30a00081a0", "fullName": "test_cases.test_verify_ph_no.TestVerifyPhoneNo#test_verify_phone_no", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('log_on_failure', 'appium_driver')"}, {"name": "tag", "value": "flaky(reruns=3)"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_verify_ph_no"}, {"name": "subSuite", "value": "TestVerifyPhoneNo"}, {"name": "host", "value": "EPINHYDW042A"}, {"name": "thread", "value": "24716-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_verify_ph_no"}]}